grails-app/views/batch/_uploadFileForm.gsp

```diff
@@ -17,7 +17,12 @@
                     <label><warehouse:message code="default.date.label"/></label>
                 </td>
                 <td class="value">
-                    <g:datePicker name="date" value="none" precision="minute" relativeYears="[-20..0]" noSelection="['':'']"/>
+                    <g:datePicker name="date"
+                                  value="none"
+                                  fieldType="${Date}"
+                                  precision="minute"
+                                  relativeYears="[-20..0]"
+                                  noSelection="['':'']"/>
```

Comment:
All these changes to gsps is adding `fieldType="${Date}"`. There is no other change (other than splitting the the attrs into multiple lines)

I apologize for the 28 file change. I decided it was better to add `fieldType="${Date}"` to all the existing date pickers (instead of adding `fieldType="${Instant}"` to all the refactored ones) because we eventually want to refactor all of these to use Instant and doing it this way means we can remove the fieldType once we've refactored a field to use Instant (as opposed to needing to remember to add `fieldType="${Instant}"` to all fields that we refactor). Once we refactor all fields then we can remove the fieldType attr and logic entirely.